extensions/mentions/less/forum.less

```
@@ -117,22 +117,30 @@
     &,
     &:hover,
     &:active {
-      color: @text-on-light;
+      color: @text-on-dark;
```

following my previous comment

```suggestion
      color: @text-on-light;
```